src/main/java/com/microsoft/aad/msal4j/AbstractClientApplicationBase.java

```diff
@@ -446,6 +456,34 @@ public T sslSocketFactory(SSLSocketFactory val) {
             return self();
         }
 
+        /**
+         * Sets the connect timeout value used in HttpsURLConnection connections made by {@link DefaultHttpClient},
+         * and is not needed if using a custom HTTP client
+         *
+         * @param val timeout value in milliseconds
+         * @return instance of the Builder on which method was called
+         */
+        public T httpClientConnectTimeout(Integer val) {
```

Comment:
name should highlight that this settings used only for default http client

Reply:
I originally called it it 'defaultHttpClientConnectTimeout' (same for the read timeout), but thought naming a settable property 'default' would be confusing. However, I now realize that a user will just think these timeouts work with the httpClient() part of the builder.

connectTimeoutForDefaultHttpClient maybe?